### PR TITLE
docs(node): Add link to alternative installation methods

### PR DESCRIPTION
**Pull request by s1gr1d**
There are different methods of correctly initializing Sentry. Added a link to a more detailed explanation of differences with ESM/CJS.

Due: 17. May

### CODE REVIEW

File: static/app/gettingStartedDocs/node/connect.tsx [hunk 1]
```diff
@@ -68,8 +69,14 @@ const onboarding: OnboardingConfig = {
         },
         {
           description: tct(
-            "Make sure to import [code1:instrument.js/mjs] at the top of your file. Set up the error handler after all controllers and before any other error middleware. This setup is typically done in your application's entry point file, which is usually [code2:index.(js|ts)].",
-            {code1: <code />, code2: <code />}
+            "Make sure to import [code1:instrument.js/mjs] at the top of your file. Set up the error handler after all controllers and before any other error middleware. This setup is typically done in your application's entry point file, which is usually [code2:index.(js|ts)]. If you are unable to import an external file, read about the [docs:different installation methods in our docs].",
```
Comment (andreiborza) [inline static/app/gettingStartedDocs/node/connect.tsx:72]:
```suggestion
            "Make sure to import [code1:instrument.js/mjs] at the top of your file. Set up the error handler after all controllers and before any other error middleware. This setup is typically done in your application's entry point file, which is usually [code2:index.(js|ts)]. If you are unable to import an external file, read about the [docs:alternate installation methods in our docs].",
```

File: static/app/gettingStartedDocs/node/express.tsx [hunk 2]
```diff
@@ -82,8 +83,14 @@ const onboarding: OnboardingConfig = {
         },
         {
           description: tct(
-            "Make sure to import [code1:instrument.js/mjs] at the top of your file. Set up the error handler after all controllers and before any other error middleware. This setup is typically done in your application's entry point file, which is usually [code2:index.(js|ts)].",
-            {code1: <code />, code2: <code />}
+            "Make sure to import [code1:instrument.js/mjs] at the top of your file. Set up the error handler after all controllers and before any other error middleware. This setup is typically done in your application's entry point file, which is usually [code2:index.(js|ts)]. If you are unable to import an external file, read about the [docs:different installation methods in our docs].",
```
Comment (andreiborza) [inline static/app/gettingStartedDocs/node/express.tsx:86]:
```suggestion
            "Make sure to import [code1:instrument.js/mjs] at the top of your file. Set up the error handler after all controllers and before any other error middleware. This setup is typically done in your application's entry point file, which is usually [code2:index.(js|ts)]. If you are unable to import an external file, read about the [docs:alternate installation methods in our docs].",
```

File: static/app/gettingStartedDocs/node/koa.tsx [hunk 3]
```diff
@@ -75,8 +76,14 @@ const onboarding: OnboardingConfig = {
         },
         {
           description: tct(
-            "Make sure to import [code1:instrument.js/mjs] at the top of your file. Set up the error handler after all controllers and before any other error middleware. This setup is typically done in your application's entry point file, which is usually [code2:index.(js|ts)].",
-            {code1: <code />, code2: <code />}
+            "Make sure to import [code1:instrument.js/mjs] at the top of your file. Set up the error handler after all controllers and before any other error middleware. This setup is typically done in your application's entry point file, which is usually [code2:index.(js|ts)]. If you are unable to import an external file, read about the [docs:different installation methods in our docs].",
```
Comment (andreiborza) [inline static/app/gettingStartedDocs/node/koa.tsx:79]:
```suggestion
            "Make sure to import [code1:instrument.js/mjs] at the top of your file. Set up the error handler after all controllers and before any other error middleware. This setup is typically done in your application's entry point file, which is usually [code2:index.(js|ts)]. If you are unable to import an external file, read about the [docs:alternate installation methods in our docs].",
```

File: static/app/gettingStartedDocs/node/node.tsx [hunk 4]
```diff
@@ -63,10 +64,17 @@ const onboarding: OnboardingConfig = {
             },
           ],
         },
+        // https://docs.sentry.io/platforms/javascript/guides/node/install
         {
           description: tct(
-            "Make sure to import [code1:instrument.js/mjs] at the top of your file. Set up the error handler after all controllers and before any other error middleware. This setup is typically done in your application's entry point file, which is usually [code2:index.(js|ts)].",
-            {code1: <code />, code2: <code />}
+            "Make sure to import [code1:instrument.js/mjs] at the top of your file. Set up the error handler after all controllers and before any other error middleware. This setup is typically done in your application's entry point file, which is usually [code2:index.(js|ts)]. If you are unable to import an external file, read about the [docs:different installation methods in our docs].",
```
Comment (andreiborza) [inline static/app/gettingStartedDocs/node/node.tsx:70]:
```suggestion
            "Make sure to import [code1:instrument.js/mjs] at the top of your file. Set up the error handler after all controllers and before any other error middleware. This setup is typically done in your application's entry point file, which is usually [code2:index.(js|ts)]. If you are unable to import an external file, read about the [docs:alternate installation methods in our docs].",
```

**Review by andreiborza — APPROVED** [review]

File: static/app/gettingStartedDocs/node/connect.tsx (hunk 1 above)
Comment (s1gr1d) [reply static/app/gettingStartedDocs/node/connect.tsx:72]: I think "different" is more fitting than "alternate" here. But I could also write: "read about alternative installation methods in our docs" 🤔

Comment (andreiborza) [reply static/app/gettingStartedDocs/node/connect.tsx:72]: I think "read about alternative installation methods" reads slightly nicer, we also call them alternate installation methods in the docs, but no strong feelings on it.

File: static/app/gettingStartedDocs/node/connect.tsx [hunk 5]
```diff
@@ -68,8 +69,14 @@ const onboarding: OnboardingConfig = {
         },
         {
           description: tct(
-            "Make sure to import [code1:instrument.js/mjs] at the top of your file. Set up the error handler after all controllers and before any other error middleware. This setup is typically done in your application's entry point file, which is usually [code2:index.(js|ts)].",
-            {code1: <code />, code2: <code />}
+            "Make sure to import [code1:instrument.js/mjs] at the top of your file. Set up the error handler after all controllers and before any other error middleware. This setup is typically done in your application's entry point file, which is usually [code2:index.(js|ts)]. If you are unable to import an external file, read about [docs:alternative installation methods in our docs].",
```
Comment (mydea) [inline static/app/gettingStartedDocs/node/connect.tsx:72]:
```suggestion
            "Make sure to import [code1:instrument.js/mjs] at the top of your file. Set up the error handler after all controllers and before any other error middleware. This setup is typically done in your application's entry point file, which is usually [code2:index.(js|ts)]. If your application runs in ESM mode, read about [docs:alternative installation methods in our docs].",
```

maybe this is clearer? I think this is the main reason you can't follow this 🤔

**Pull request merged**